Source/analyzer/item.cpp

```diff
@@ -122,7 +122,7 @@ static void DropAllItems()
 
 bool ScannerItem::skipLevel(int level)
 {
-	if (level == 0)
+	if (*Config.target == 0 && level >= 0 && level <= 16)
```

Comment:
If looking for town and current level is validt? Should there have been an and somewhere? Will this become an issue when hellfire support might be added?